src/main/java/io/wispforest/owo/client/OwoClient.java

```diff
@@ -74,5 +76,10 @@ public void onInitializeClient() {
 
         if (!Owo.DEBUG) return;
         OwoDebugCommands.Client.register();
+
+        ResourceManagerHelper.get(ResourceType.CLIENT_RESOURCES).registerReloadListener(new SimpleSynchronousResourceReloadListener() {
+            @Override public Identifier getFabricId() { return Identifier.of("owo", "after_shader_load"); }
+            @Override public void reload(ResourceManager manager) { GlProgram.forEachProgram(Runnable::run); }
+        });
```

Comment:
won't this only run in debug mode?

Reply:
Arg really did not see the above if return...